### ISSUE TITLE
We should not enable rnode and node by default

### ISSUE THREAD

**Issue by ericfranz** [OSC/ood-portal-generator#4]
The default installation of OOD should not have these enabled. This should be an opt in, with the admin installing OOD understanding exactly what she is doing when enabling this.

**Comment by basilgohar** [comment]
@ericfranz I'm not opposed to this.  I think our defaults will evolve as the product matures and more-and-more features can be enabled by default as the product becomes more stable, so removing things right now is fine by me.

**Comment by nickjer** [comment]
Just be aware that this is a core feature of Open OnDemand that we advertised (being able to proxy to backend nodes).

https://osu.box.com/s/8rne1z5nogp6hksyjxb76d83k6tttolw

Is the reason for making this opt-in due to a security concern? If so, then I can disable it by default, but we would also need to add appropriate documentation in this README as well as in the OOD installation manual clearly informing the sys admin that the OOD portal will be missing features unless this feature is explicitly enabled. I may also have to update the VNCSim app installation warning the sys admin that this feature is a requirement.

**Comment by ericfranz** [comment]
Security concern. Currently we don't advertise well the features of Open OnDemand => the basic overview of what you get after you have installed the product.

Most of the previews have been regarding the file explorer and the shell access apps. Following the default installation, they might be surprised that an authenticated user can now access remotely any remote server running on their systems.

So in other words, I think it should be opt in, just like it is opt in to install a file explorer or shell app etc. There should also be a clear section in the directions in the README to enable this feature during installation or how to enable it to an existing OOD installation.

**Comment by nickjer** [comment]
Tasks:

  - [ ] Add proxying to backend nodes an opt-in feature
  - [ ] Update `README.md` detailing this
  - [ ] Update Open OnDemand installation directions detailing this as well
  - [ ] Update VNCSim app documentation detailing the requirement of this feature

**Comment by basilgohar** [comment]
@nickjer I think we don't need to have all the features we promised in every single release before the final one, so it's okay to revert the feature if we don't make use of it currently, in my opinion.

**Comment by nickjer** [comment]
@basilgohar: What do you mean when you say "revert the feature"?

I was under the impression that we are just disabling it by default, but still allowing for it to be enabled by a sys admin if they so choose. Is this assumption correct?
